code/game/objects/items/weapons/RSF.dm

```diff
@@ -1,116 +1,366 @@
+//Reworked and refactored by Pixelnator on 20.3.2014 to support printing items directly into peoples hands provided they accept the give prompt.
+//Updated by Pixelnator on 12.8.2015 to work with Unity2.0 code.
+
 /*
 CONTAINS:
 RSF
-
 */
-
 /obj/item/weapon/rsf
-	name = "\improper Rapid-Service-Fabricator"
```

Comment:
\improper is used for some parts of the naming code, mostly for including "The XXX" and for some forms of areas. Please re-add it